sw/device/silicon_creator/lib/drivers/flash_ctrl.h

```diff
@@ -79,15 +79,15 @@ typedef enum flash_ctrl_partition {
   X(kFlashCtrlInfoPageOwnerReserved1,      0, 6) \
   X(kFlashCtrlInfoPageOwnerReserved2,      0, 7) \
   X(kFlashCtrlInfoPageOwnerReserved3,      0, 8) \
-  X(kFlashCtrlInfoPageCreatorReserved0,    0, 9) \
+  X(kFlashCtrlInfoPageFactoryCerts,        0, 9) \
   /**
    * Bank 1 information partition type 0 pages.
    */ \
   X(kFlashCtrlInfoPageBootData0,           1, 0) \
   X(kFlashCtrlInfoPageBootData1,           1, 1) \
   X(kFlashCtrlInfoPageOwnerSlot0,          1, 2) \
   X(kFlashCtrlInfoPageOwnerSlot1,          1, 3) \
-  X(kFlashCtrlInfoPageCreatorReserved1,    1, 4) \
+  X(kFlashCtrlInfoPageCreatorReserved0,    1, 4) \
```

Comment:
why does this have to be renamed?

Reply:
Because `kFlashCtrlInfoPageCreatorReserved0` was consumed so we should start from 0 again.